### ISSUE TITLE
expected loss value on custom dataset

### ISSUE THREAD

**Issue by chinmay5** [facebookresearch/mae#16]
Thank you so much for releasing the code.

I wanted to train the model from scratch on a custom dataset. I was wondering if you could share what was the loss value computed for the pre-training step. I was getting a very high loss value and was wondering what could be the possible reason. 
![image](https://user-images.githubusercontent.com/16525717/149550831-1be9fc87-22ca-4e61-8657-f49388b73873.png)

Also, is there some correlation between the amount of masking and the reconstruction loss? Can a high loss value indicate that the masking ratio is low?

**Comment by KaimingHe** [comment]
The loss is ~0.3 on ImageNet when reconstructing pixels. I recommend you to run our ImageNet training code as a sanity check to make sure you can run the code correctly. The masking ratio should impact the reconstruction loss.

**Comment by mts42000** [comment]
Is 0.3 for the pixel reconstruction without norm or with `norm_pix_loss`?
